tests/foreman/api/test_contentview.py

```diff
@@ -428,6 +428,48 @@ def test_positive_promote_3(self):
         self.assertEqual(len(cvv_attrs['environments']), REPEAT + 1)
         self.assertEqual(len(cvv_attrs['puppet_modules']), 1)
 
+    def test_add_normal_cv_to_composite(self):
+        """@Test: Create normal content view, publish and
+        add it to a new composite content view
+
+        @Assert: Content view can be created and assigned to
+        composite one through content view versions
+        mechanism
+
+        @Feature: ContentView
+
+        """
+        content_view = entities.ContentView(organization=self.org.id)
+        content_view.id = content_view.create_json()['id']
+        content_view.set_repository_ids([self.yum_repo.id])
+        content_view.publish()
+        for cvv_id in (
+                version['id']
+                for version
+                in content_view.read_json()['versions']):
+            entities.ContentViewVersion(id=cvv_id)
```

Comment:
Creating a `ContentViewVersion` object and doing nothing with that object is a no-op. Can this be simplified?